planner/core/plan.go

```diff
@@ -312,7 +312,8 @@ func (p *basePhysicalPlan) SetChild(i int, child PhysicalPlan) {
 	p.children[i] = child
 }
 
-func (p *basePlan) context() sessionctx.Context {
+// Context implements Plan Context interface.
+func (p *basePlan) Context() sessionctx.Context {
```

Comment:
`Context` is a little confusion, how about renaming it to `SCtx()` to be consist with other places?